truenas/webui
# NAS-111392 / 21.08 / NAS-111392 - Fix SMART tests page

Testing: 
1. Find an instance that supports smart testing (https://10.238.15.92)
2. Go to Storage -> Disks
3. Ensure that existing smart tests appear correctly for disks that have them.
4. Ensure that there is no broken Add test result in list of results.
